addons/binding/org.openhab.binding.camera/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="camera"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+    <name>Camera Binding</name>
+    <description>Binding for generic IP cameras.</description>
```

Comment:
Would the correct description be: 'Generic binding for IP cameras.'

Reply:
OK